include/actor_types.h

```diff
@@ -69,27 +69,27 @@ exactly what you're doing.
 #define ACTOR_IS_NOT_EXPIRED 0xF // The actor possesses some kind of collision and can be removed
 
 // Actor shell->state (green, red and blue)
-#define HELD_SHELL 0 // A shell is being held by the player with Z.
-#define RELEASED_SHELL 1 // The shell is in the processed of being shot. (Only applies to single shells.)
-#define MOVING_SHELL 2 // The shell has been shot and is now in a moving state where it goes forwards.
-#define RED_SHELL_LOCK_ON 3 // A red shell has found a target and is semi hastily appraching it.
-#define A_TRIPLE_GREEN_SHELL 4 // A green triple shell. Once it's shot it's of course no longer a triple shell.
+#define HELD_SHELL 0 // Single shell that has not been dropped. (probably holding Z).
+#define RELEASED_SHELL 1 // This is the short window where single shells aren't being held or launched.
+#define MOVING_SHELL 2 // Moving towards its target after being shot.
+#define RED_SHELL_LOCK_ON 3 // Red shell is targeting.
+#define A_TRIPLE_GREEN_SHELL 4 // Loses triple shell state when shot.
```

Comment:
How do you feel about `TRIPLE_GREEN_SHELLS` instead of `A_` (same for triple red sheets)

Reply:
Yeah